### PR TITLE
chore: prepare ivy compatible build

### PR DESCRIPTION
**Pull request by androizer**
Seem like `"compilationMode": "partial" in tsconfig.prod.json` is important for angular to fallback when ivy compatible application is not found.

More: https://angular.io/guide/creating-libraries#publishing-libraries

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 # dependencies
 /node_modules
+/**/node_modules
```
Comment (midgleyc) [inline .gitignore]:
```suggestion
node_modules
```

alone should cover both cases

File: .gitignore [hunk 2]
```diff
@@ -9,6 +9,7 @@
...
```
Comment (midgleyc) [inline .gitignore:12]: you also might want to add `.angular`, which is a cache on Angular 13.

File: package.json [hunk 3]
```diff
@@ -7,7 +7,8 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e",
+    "clean": "rimraf node_modules"
   },
   "private": true,
   "dependencies": {
```
Comment (midgleyc) [inline package.json:14]: You still have references to Angular 11 here. I think these need to be at least 12 before the build will use Ivy successfully.

File: projects/ngx-breadcrumb/tsconfig.lib.prod.json [hunk 4]
```diff
@@ -5,6 +5,7 @@
     "declarationMap": false
   },
   "angularCompilerOptions": {
-    "enableIvy": false
+    "enableIvy": true,
```
Comment (midgleyc) [inline projects/ngx-breadcrumb/tsconfig.lib.prod.json]: By https://v12.angular.io/guide/creating-libraries#publishing-libraries you should be able to omit `enableIvy` entirely, and just have `compilationMode`.

**Review by midgleyc — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (androizer) [reply .gitignore]: Done.

File: .gitignore (hunk 2 above)
Comment (androizer) [reply .gitignore:12]: Added.

File: package.json (hunk 3 above)
Comment (androizer) [reply package.json:14]: Yes, you are right. the angular CLI depends upon `ng-packager`, which is now bumped to angular v12 compliant version too.

File: projects/ngx-breadcrumb/tsconfig.lib.prod.json (hunk 4 above)
Comment (androizer) [reply projects/ngx-breadcrumb/tsconfig.lib.prod.json]: `enableEvy` removed.

**Review by midgleyc — APPROVED** [review]

**Comment by midgleyc** [comment]
I think this should be good to go now.

**Pull request merged**